slidey_js.js

```diff
@@ -0,0 +1,25 @@
+$(document).ready(function(){
+
+var images = ["http://i.imgur.com/wcbCIc3.jpg", "http://i.imgur.com/2tQTR7t.jpg","http://i.imgur.com/ejbGEh2.gif", "http://i.imgur.com/u0tOBYz.jpg", "http://i.imgur.com/VATK187.jpg"];
+
+var backer = $("#background");
+
+var i = 0;
+
+var changeImage = function(){
+	startOver();
+	backer.css("background-image", "url("+ images[i] + ")").fadeIn(600).delay(1800).fadeOut(600); i++;
```

Comment:
this looks good! You could also use a callback for the `fadeIn()` function here instead of using a delay, but both work just as well.

Actually, I think you need a delay anyway to show one slide for a set amount of time. Nice job!